index.html

```diff
@@ -1,13 +1,570 @@
+            9?
+          </h3>
+          <h6>By Drew McLellan</h6>
+          <p>
+            If text is set ragged right, the word space (the space between
+            words) can be fixed and unchanging. If the text is justified (set
+            flush left and right), the word space must be elastic.
+          </p>
+          <p class="arrow-2">
+            <i class="fas fa-2x fa-long-arrow-alt-right"></i>
+          </p>
+        </article>
+      </div>
+    </div>
+
+    <div class="grid-three box">
+      <div class="col-1a midium-gray-bg">
+        <article class="article-2">
+          <p>3 Days Ago</p>
+          <h3>
+            Smashing Podcast Episode 17 With Angie Bryon: What's New In Drupal
+            9?
+          </h3>
+          <h6>By Drew McLellan</h6>
+          <p>
+            If text is set ragged right, the word space (the space between
+            words) can be fixed and unchanging. If the text is justified (set
+            flush left and right), the word space must be elastic.
+          </p>
+          <p class="arrow-2">
+            <i class="fas fa-2x fa-long-arrow-alt-right"></i>
+          </p>
+        </article>
+      </div>
+      <div class="col-2a"></div>
+      <div class="col-3a dark-gray-bg"></div>
+    </div>
+
+    <div class="latest-articles light-gray-bg">
+      <p><a href="#">SEE ALL THE LATEST ARTICLES →</a></p>
+    </div>
+
+
+  </main>
+  <footer class="dark-gray-bg">
+    <div class="ft-section">
+      <p>BROWSE ALL SMASHING MAGAZINE TOPICS</p>
+
+      <ul class="ul-ft">
+      <li><a href="#">Accessibility</a></li>
+      <li><a href="#">Androids</a></li>
+      <li><a href="#">Animation</a></li>
+      <li><a href="#">Apps</a></li>
+      <li><a href="#">CSS</a></li>
+      <li><a href="#">Design</a></li>
+      <li><a href="#">Design Patterns</a></li>
+      <li><a href="#">Design Systems</a></li>
+      <li><a href="#">E-Commerce</a></li>
+      <li><a href="#">Freebies</a></li>
+      </ul>
+
+      <ul class="ul-ft">
+      <li><a href="#">Accessibility</a></li>
+      <li><a href="#">Androids</a></li>
+      <li><a href="#">Animation</a></li>
+      <li><a href="#">Apps</a></li>
+      <li><a href="#">CSS</a></li>
+      <li><a href="#">Design</a></li>
+      <li><a href="#">Design Patterns</a></li>
+      <li><a href="#">Design Systems</a></li>
+      <li><a href="#">E-Commerce</a></li>
+      <li><a href="#">Freebies</a></li>
+      </ul>
+
+      <ul class="ul-ft">
+      <li><a href="#">Accessibility</a></li>
+      <li><a href="#">Androids</a></li>
+      <li><a href="#">Animation</a></li>
+      <li><a href="#">Apps</a></li>
+      <li><a href="#">CSS</a></li>
+      <li><a href="#">Design</a></li>
+      <li><a href="#">Design Patterns</a></li>
+      <li><a href="#">Design Systems</a></li>
+      <li><a href="#">E-Commerce</a></li>
+      <li><a href="#">Freebies</a></li>
+      </ul>
+
+  </div>
+   
+  <div class="sub-footer">
```

Comment:
The footer must have two placeholders for the two images on both sides.
Original 👇
![Annotation 2020-06-06 180247](https://user-images.githubusercontent.com/57726348/83944285-f7056580-a81f-11ea-92c9-26c36930b953.jpg)

Right now 👇
![Annotation 2020-06-06 180209](https://user-images.githubusercontent.com/57726348/83944294-02589100-a820-11ea-85d5-380a08f5b730.jpg)

Reply:
Thanks for the review. Have updated the two images in the footer section. Please note that the smashing magazine images are changing from time to time as the site is opened. We have updated two images to demonstrate that the images can be positioned in that position.